src/screens/EventDetailsScreen/index.tsx

```diff
@@ -73,13 +73,25 @@ const EventDetailsScreen: React.FC<TEventDetalsScreenProps> = ({ route }) => {
       <Player
         autoPlay
         configuration={{
-          url: 'https://video-ingestor-output-bucket.s3-eu-west-1.amazonaws.com/12338/manifest.m3u8',
+          url: 'https://video-ingestor-output-bucket.s3.eu-west-1.amazonaws.com/6565/manifest.m3u8', // https://video-ingestor-output-bucket.s3-eu-west-1.amazonaws.com/12338/manifest.m3u8
```

Comment:
As above

Reply:
it is a WIP, about subtitles. I will remove it later.